### PR TITLE
Update docker-compose and remove tox

### LINKED ISSUE #141: Upgrade django version
```
Traceback (most recent call last):
  File "/tmp/tmpZYrl8_.py", line 14, in <module>
    main()
  File "/tmp/tmpZYrl8_.py", line 10, in main
    cli([os.path.join(PROJECT_DIRECTORY, 'requirements.in')])
  File "/home/nathan/.virtualenvs/cc/local/lib/python2.7/site-packages/click/core.py", line 722, in __call__
    return self.main(*args, **kwargs)
  File "/home/nathan/.virtualenvs/cc/local/lib/python2.7/site-packages/click/core.py", line 697, in main
    rv = self.invoke(ctx)
  File "/home/nathan/.virtualenvs/cc/local/lib/python2.7/site-packages/click/core.py", line 895, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/home/nathan/.virtualenvs/cc/local/lib/python2.7/site-packages/click/core.py", line 535, in invoke
    return callback(*args, **kwargs)
  File "/home/nathan/.virtualenvs/cc/local/lib/python2.7/site-packages/piptools/scripts/compile.py", line 184, in cli
    results = resolver.resolve(max_rounds=max_rounds)
  File "/home/nathan/.virtualenvs/cc/local/lib/python2.7/site-packages/piptools/resolver.py", line 107, in resolve
    has_changed, best_matches = self._resolve_one_round()
  File "/home/nathan/.virtualenvs/cc/local/lib/python2.7/site-packages/piptools/resolver.py", line 195, in _resolve_one_round
    for dep in self._iter_dependencies(best_match):
  File "/home/nathan/.virtualenvs/cc/local/lib/python2.7/site-packages/piptools/resolver.py", line 274, in _iter_dependencies
    dependencies = self.repository.get_dependencies(ireq)
  File "/home/nathan/.virtualenvs/cc/local/lib/python2.7/site-packages/piptools/repositories/pypi.py", line 145, in get_dependencies
    self._dependencies_cache[ireq] = reqset._prepare_file(self.finder, ireq)
  File "/home/nathan/.virtualenvs/cc/local/lib/python2.7/site-packages/pip/req/req_set.py", line 554, in _prepare_file
    require_hashes
  File "/home/nathan/.virtualenvs/cc/local/lib/python2.7/site-packages/pip/req/req_install.py", line 278, in populate_link
    self.link = finder.find_requirement(self, upgrade)
  File "/home/nathan/.virtualenvs/cc/local/lib/python2.7/site-packages/pip/index.py", line 514, in find_requirement
    'No matching distribution found for %s' % req
pip.exceptions.DistributionNotFound: No matching distribution found for django==2.0.8
ERROR: Stopping generation because post_gen_project hook script didn't exit successfully
Hook script failed (exit status: 1)
```

### LINKED ISSUE #144: Remove tox
https://github.com/mitodl/mitxpro/pull/44

### PR DESCRIPTION
**Pull request by noisecapella**
Fixes #141 
Fixes #144

### CODE REVIEW

**Comment by noisecapella** [comment]
@gsidebo Any comments on this?

**Comment by gsidebo** [comment]
Looking now. Sorry about that!

**Comment by gsidebo** [comment]
@noisecapella I'm getting this message when trying to install from `requirements.txt` in a pyenv: `ERROR: docker-compose 1.14.0 has requirement requests!=2.11.0,<2.12,>=2.6.1, but you'll have requests 2.23.0 which is incompatible.`.

File: {{ cookiecutter.project_name }}/test_requirements.txt [hunk 1]
```diff
@@ -1,5 +1,4 @@
-astroid==2.0.4
-black==18.9b0
+black
```
Comment (gsidebo) [inline {{ cookiecutter.project_name }}/test_requirements.txt]: If we're adding the `black` dependency, should we also add a `pyproject.toml` file like xpro has? https://github.com/mitodl/mitxpro/blob/master/pyproject.toml

Comment (noisecapella) [reply {{ cookiecutter.project_name }}/test_requirements.txt]: Ah, good point

**Comment by noisecapella** [comment]
Comment addressed

**Pull request merged**